dapr/clients/grpc/_response.py

```diff
@@ -661,6 +664,31 @@ def items(self) -> Sequence[ConfigurationItem]:
         return self._items
 
 
+class ConfigurationWatcher():
+    def __init__(self):
+        self.items = []
+
+    def get_items(self):
+        return self.items
+
+    def watch_configuration(self, stub, store_name, keys, config_metadata):
+        req = api_v1.SubscribeConfigurationRequest(
+            store_name=store_name, keys=keys, metadata=config_metadata)
+        thread = threading.Thread(target=self.read_subscribe_config, args=(stub, req))
+        thread.daemon = True
+        thread.start()
+
+    def read_subscribe_config(self, stub, req):
```

Comment:
@berndverst Do you think this should be `_read_subscribe_config` just to let users know it is an internal method and should not be used outside the class?

Reply:
@berndverst thoughts on this?

Reply:
I agree. @amulyavarote, can you update that? Private / internal method should begin with an underscore `_`